nephos/fabric/utils.py

```diff
@@ -105,6 +105,83 @@ def get_helm_pod(namespace, release, app, item=0):
     return get_pod(namespace, identifier, item=item)
 
 
+def get_org_tls_ca_cert(opts, msp_namespace):
+    """Get path to the directory containing tls CA certificate
+
+        Args:
+            opts (dict): Nephos options dict.
+            msp_name (str): Name of Membership Service Provider.
+
+        Returns:
+            path: path to the directory containing tls CA certificate
+    """
+
+    if "tls_ca" in opts["ordering"]["tls"]:
+        glob_target = f"{opts['core']['dir_crypto']}/tlscacerts/*.crt"
+    else:
+        glob_target = f"{opts['core']['dir_crypto']}/crypto-config/*Organizations/{msp_namespace}*/tlsca/*.pem"
+    tls_path_list = glob(glob_target)
+    if len(tls_path_list) == 1:
+        return tls_path_list[0]
+    else:
+        raise ValueError(
+            f"TLS path list length is {len(tls_path_list)} - {tls_path_list}"
+        )
+
+
+def get_tls_path(opts, id_type, namespace, release):
+    """Get path to the directory containing tls materials for a node
+        Args:
+            opts (dict): Nephos options dict.
+            id_type (str): Type of ID we use.
+            namespace (str): Name of namespace.
+            release (str): Name of release/node.
+
+        Returns:
+            path: path to the directory containing materials of a node
+    """
+
+    if "tls_ca" in opts["ordering"]["tls"]:
+        glob_target = f"{opts['core']['dir_crypto']}/{release}_TLS/tls"
+    else:
+        glob_target = f"{opts['core']['dir_crypto']}/crypto-config/{id_type}Organizations/{namespace}*/{id_type}s/{release}*/tls"
+    tls_path_list = glob(glob_target)
+    if len(tls_path_list) == 1:
+        return tls_path_list[0]
+    else:
+        raise ValueError(
+            f"MSP path list length is {len(tls_path_list)} - {tls_path_list}"
+        )
+
+
+def is_orderer_tls_true(opts):
+    """Check if tls is enabled for orderer
+        Args:
+            opts (dict): Nephos options dict.
+
+        Returns:
+            Boolean: return true if tls is enabled for orderer and false otherwise
```

Comment:
Boolean in python is "bool" so:

"bool: Return true if TLS is enabled for orderer and false otherwise"